cockatrice/src/chatview.cpp

```diff
@@ -374,9 +417,27 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 while (fullMentionUpToSpaceOrEnd.size());
             }
         }
+        else if (index == highlightWordFirstIndex)
+        {
+            // You have received a valid mention of custom word!!
+            int firstSpace = (message.indexOf(" ") == -1 ? message.size() : message.indexOf(" "));
+            mentionFormat.setBackground(QBrush(getCustomMentionColor()));
```

Comment:
Do we want these to use the same color as a mention? I think it should either a different color or able for the user to set it. Mention color is specific to you and your username, not to custom words. Ideally I would like to set a different color for different words.